instagram/src/components/PostContainer/PostsPage.js

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import PostContainer from './PostContainer'
+import { dummyData } from '../../data/dummy-data'
+import SearchContainer from '../SearchContainer/SearchContainer'
+import './PostContainer.css'
+
+class PostsPage extends React.Component{
+    constructor(props){
+        super(props);
+        this.state = {
+            data:[],
+            newComment:'',
+            likes:'',
+            searchText:''
+        }
+    }
+    handleSearchChange = event =>{
+        this.setState({searchText:event.target.value})
+    }
+    handleSearchSubmit = event =>{
+        event.preventDefault();
+        this.setState({data:this.state.data.filter(element=>{
```

Comment:
be warned, this is overwriting your data each time you search, so you wouldn't get it back when you remove your search filter